content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -61,6 +61,20 @@ among the comma-delimited, ordered list of values for the `--enable-admission-pl
 the API server component. For more information on API server command-line flags,
 check [kube-apiserver](/docs/admin/kube-apiserver/) documentation.
 
+Claims can also be created without any StorageClass while no default exists. In that case
+their StorageClass will remain unset until a default becomes available, then the claim
+will get the new default assigned by `RetroactiveDefaultStorageClass` [feature](/docs/concepts/storage/persistent-volumes/#retroactive-storageclass-assignment).
+
+### Retroactive StorageClass assignment
```

Comment:
```suggestion
### Retroactive StorageClass assignment

You can create a PersistentVolumeClaim without specifying a `storageClassName` for the new
PVC, and you can do so even when no default StorageClass exists in your cluster.
In this case, the new PVC creates as you defined it, and the `storageClassName` of that PVC
remains set to `""` until a default becomes available. If or when you or somebody else marks a
StorageClass as default for your cluster, that PVC continues to have the `storageClassName: ""`.
However, if you enable the `RetroactiveDefaultStorageClass` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) then Kubernetes behaves differently: existing PVCs that have an
empty `storageClassName` update to use the new default StorageClass.
```

Reply:
It seems that this file combines both 1st and 3rd person forms, however the surrounding articles are in the 3rd person form so I tried to maintain that. Maybe there is some rule related to this?

Also I think the first form can be confusing, because sometimes articles describe operations that are typically done by administrator of the cluster (with elevated priviliges) and operations done by a regular user. For example in case of classes and volumes, a class would be created by admin and PVC by user.

Lastly the sentence suggested:

"You can create a PersistentVolumeClaim without specifying a `storageClassName`...

...In this case, the new PVC creates as you defined it, and the `storageClassName` of that PVC remains set to `""` until a default becomes available."

It's not very accurate, you don't magically get `""` when you don't specify class name in PVC. Because "without specifying" can mean either leaving out `storageClassName` key out or setting it to nil value or `""`. All these are allowed when creating PVCs and it's important to distinguish those because historically this was the same thing but not anymore. Only the case with `""` truly means that user wants no class. This is described somewhere around L74.

Reply:
> it's important to distinguish those because historically this was the same thing but not anymore

Let's call that out. It wasn't obvious to me.

Reply:
Ack, moved under "Retroactive StorageClass assignment" and changed to active form.